### ISSUE TITLE
[RFE] mailman-web should include django-auth-ldap

### ISSUE THREAD

**Issue by stettberger** [maxking/docker-mailman#374]
**Tag/version of Container Images**: latest

As I want LDAP user login, as described [here](https://gitlab.com/mailman/postorius/issues/132), I required to derive my own Docker container from mailman-web:latest. It would be great if that could already be included. I used the follwing Dockerfile:

```
FROM maxking/mailman-web:latest
RUN set -ex \
        && apk add --no-cache --virtual .build-deps \
             gcc libc-dev python3-dev libffi-dev openldap-dev \
        && python3 -m pip install  python-ldap django-auth-ldap \
        && apk del .build-deps
```

**Comment by maxking** [comment]
Can you open a PR to edit the `Dockerfile` and `Dockerfile.dev` to add these two packages?

**Comment by maxking** [comment]
Fixed by #375
